tests/unit/PolicyUtilsTest.ts

```diff
@@ -81,4 +91,57 @@ describe('PolicyUtils', () => {
             });
         });
     });
+    describe('getActivePolicies', () => {
+        beforeAll(() => {
+            Onyx.init({
+                keys: ONYXKEYS,
+                initialKeyStates: {
+                    // Given mock data for session
+                    [ONYXKEYS.SESSION]: {accountID: CARLOS_ACCOUNT_ID, email: CARLOS_EMAIL},
+                },
+            });
+        });
+
+        beforeEach(() => {
+            global.fetch = TestHelper.getGlobalFetchMock();
+            return Onyx.clear().then(waitForBatchedUpdates);
+        });
+        it('should return empty array', () => {
+            // Given mock data for policies, contains only one workspace does not has role prop.
```

Comment:
```suggestion
            // Given a user with a single archived paid policy
```
This is clearer as it describes the actual "real-world" condition, rather than in terms of what props are set.